src/puppetlabs/puppetdb/amq_migration.clj

```diff
@@ -329,28 +346,35 @@
         mq-discard-dir (str (io/file mq-dir "discard"))
         mq-endpoint (:endpoint mq-config default-mq-endpoint)
         process-message (create-message-processor enqueue-fn dlo)
-        id (atom 0)]
+        id (atom 0)
+        fail-msg "Unable to receive ActiveMQ messages. Migration of existing messages failed."]
```

Comment:
Would it be better to say "Some ActiveMQ messages failed to transfer" here? The current error sort of gives the impression that nothing happened, which isn't necessarily the case.